api/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/api/service/HenvendelseUtsendingService.java

```diff
@@ -7,7 +7,7 @@
 import java.util.Optional;
 
 public interface HenvendelseUtsendingService {
-    void sendHenvendelse(Melding melding, Optional<String> oppgaveId, Optional<Sak> sak, String saksbehandlersValgteEnhet) throws Exception;
+    HenvendelseStatus sendHenvendelse(Melding melding, Optional<String> oppgaveId, Optional<Sak> sak, String saksbehandlersValgteEnhet) throws Exception;
```

Comment:
Ville gått for `Either<String, String>`, evt `Try<Void>` siden vi egentlig ikke bryr oss om ok-verdien